src/console_cmds.cpp

```diff
@@ -614,6 +615,115 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
+{
+	const char* const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	IniFile ini { group_names };
+	ini.LoadFromDisk(path, NO_DIRECTORY);
+	const IniGroup *ban_group = ini.GetGroup("bans", false);
+
+	if (ban_group == nullptr) return -1;
+
+	int loaded_count = 0;
+	for (const IniItem *item = ban_group->item; item != nullptr; item = item->next) {
+		// banned clients are added to global ban list when kicked or banned
```

Comment:
This comment is misleading. Kicking does not put clients on the ban list. Maybe something along the lines of:
`// add IP address to the ban list and kick clients using that IP address`

Reply:
If we're at coding style, it also uses the wrong comment type (should be `/* */`).